imports/plugins/core/core/server/Reaction/core.js

```diff
@@ -436,7 +436,7 @@ export default {
     // in cases where the domain could match multiple shops, we first check
     // whether the primaryShop matches the current domain. If so, we give it
     // priority
-    if (primaryShop.domains.includes(domain)) {
+    if (primaryShop && primaryShop.domains.includes(domain)) {
```

Comment:
Maybe also add `&& Array.isArray(primaryShop.domains)` just to be sure

Reply:
@aldeed Done!